entrypoint.sh

```diff
@@ -32,11 +32,13 @@ fi
 DEFAULT_PORT=$RANDOM
 let "DEFAULT_PORT += 32768"
 
+
```

Comment:
Check if `SSH_BIND_IP` is set.  If it is set, print a WARNing that additional server configuration may be required and  point to the documentation.